### PR TITLE
Update readme and add examples

### PR DESCRIPTION
**Pull request by mariusmihaic**
- Small update to `README`
- Added `examples.md` which contains examples with some of the current futures
- Fix in `gitignore` to stop ignoring every new file

### CODE REVIEW

File: examples/examples.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+    }
+    
+}
+catch (...)
+{
+    // ...
+}
+```
+
+## 1.2 Examples. CLI
```
Comment (bogdan-rosianu) [inline examples/examples.md]: `## 2.2. Examples. CLI`

File: examples/examples.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+|-----------------|--------------------------------------------------------------------|
+| **Address**     | Create, compute bech32 address, get public key from bech32 address |
+| **Key File**    | Read pem and json + password file                                  |
+| **Signer**      | Sign any string message or ERD prefixed messages                   |
+| **Transaction** | Create, Sign, Serialize, Deserialize                               |
+| **Proxy**       | Get account, send transaction, get ESDT balances                   |
+| **ESDT**        | Issue, transfer, transfer with SC call                             |
+
+
+## 1.1 Examples. SDK
```
Comment (bogdan-rosianu) [inline examples/examples.md]: why 1.1 ? could have been 2, so `## 2.1. Examples. SDK`

Comment (bogdan-rosianu) [reply examples/examples.md]: also add `# 2. Examples` where you explain what the examples will be focused on

**Review by sstanculeanu — DISMISSED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

File: examples/examples.md (hunk 1 above)
Comment (mariusmihaic) [reply examples/examples.md]: Agree, would be better. Refactored as suggested.

**Review by sstanculeanu — APPROVED** [review]

**Pull request merged**